noxfile.py

```diff
@@ -108,11 +110,22 @@ def start_integration_test_environment(session: Session):
 
 @nox.session(python=False)
 def build_language_container(session: Session):
-    script_path = ROOT_DIR / "build_language_container.sh"
-    session.run(str(script_path))
+    # Is this path OK?
+    # Which artifacts need to be included in releases of AAF?
+    # How should these artifacts be included?
+    # Just by uploading and adding to the GitHub release?
```

Comment:
For the AAF, we only need the release in pypi for the moment, because it is currently only used as library. If this changes, we can release the SLC on GitHub.

Reply:
Should we specify the export_path or use the temporary export_path generated by `LanguageContainerBuilder`?

Reply:
What does temporary mean? If it is really temporary, that it will be gone afterward, so it doesn't bring anything

'

Reply:
PEC [LanguageContainerBuilder](https://github.com/exasol/python-extension-common/blob/main/exasol/python_extension_common/deployment/language_container_builder.py#L55) uses 

```python
self._root_path = Path(tempfile.mkdtemp())
```
and method [export()](https://github.com/exasol/python-extension-common/blob/main/exasol/python_extension_common/deployment/language_container_builder.py#L120) either expects an argument `export_path` or uses 
```
if not export_path:
    export_path = self._root_path / '.export'
```
I see the following options:
* O1) We specify an export path within the directory of AAF
* O2) User must specify the export path when calling nox task `build_language_container`
* O3) AAF uses the default behavior of PEC, which results in a location below `tempfile.mkdtemp()`

Reply:
O3 is no option, because then the exported file is gone after the export and the nox session is anyway for interactive use or the CI at some point. For the nox session we should defne an export path within the directory of AAF

Reply:
Many thanks.
I implemented option O1 with path `.slc` within the directory of AAF.